src/lib/parts.ts

```diff
@@ -121,16 +121,11 @@ export class AttributeCommitter {
       // Next breaking change, consider making this param required.
       templatePart?: AttributeTemplatePart,
       kind: 'property'|'attribute' = 'attribute') {
+    const fqn = getAttrName(name);
```

Comment:
Prefer immediate destructuring in this sort of case:

```typescript
const [namespace, name] = getAttrName(name);
```

Reply:
This transpiles down to really icky code, so I usually avoid it whenever possible. I don't think there's any other use of iterator destructuring in the codebase yet. Maybe we should return an object instead, and object destructure that?

Reply:
Ah, fair enough, I forget it's iterator destructuring and not array destructuring. This is fine as is, but please move this const down right next to lines where it's used.

Reply:
Done.